apps/sertra/pages/server/[id].tsx

```diff
@@ -66,16 +92,15 @@ const ServerDetail: React.FC<{ detail_listing: ServerListingDetailData }> = ({
           <section>
             <h2 className={styles.sectionTitle}>Mods</h2>
             <div>
-              {/* TODO: So we are missing the name, description and artifacts, from the API data */}
-              {/* {data.mods.map((x) => (
+              {detail_listing.mods.map((x) => (
                 <ModCard
                   key={x.name}
                   name={x.name}
-                  description={x.description}
+                  owner={x.owner}
```

Comment:
When object keys nicely match component props, you can do something like this:
`<ModCard key={x.name} {...x} description={x.description ?? null} />`

`{...x}` will override any props defined before it with values from `x`. Since there's no `x.key`, the key prop won't be overwritten.
Props defined after `{...x}` will override any props defined by `{...x}`. So even though there's `x.description`, redefining the prop will overwrite the value received from `x`. **Note**: I can't recall if this works here since the type of `x.description` might not match the type of `ModCard.description`. Worth checking out anyway, or at least for you to know in the future (in case you did not know this already).

Reply:
Seems like `<ModCard key={x.name} {...x} />` does work here, just had to define type for `x` correctly. Didn't know this way was possible. Definitely is a lot simpler and cleaner representation.

Reply:
Yeah, it's simpler, but use your own judgement when to use it. Since it obfuscates the props that are actually passed (or left out), it might be a bit confusing in some situations.

Also probably `props` or `modProps` would be a better variable name than `x` now that there's not that much repetition, but that's just an opinion.

Mark this comment resolved once you've read it.

Reply:
Changed the name 👍 And good point, I'll have to keep it mind, when using some more complex component.